[2021-07-20 17:39:40,450] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Fuse.fuse6 2021-07-19T00:02:00+00:00 [queued]>
[2021-07-20 17:39:40,460] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Fuse.fuse6 2021-07-19T00:02:00+00:00 [queued]>
[2021-07-20 17:39:40,460] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-20 17:39:40,460] {taskinstance.py:1068} INFO - Starting attempt 1 of 8
[2021-07-20 17:39:40,460] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-20 17:39:40,540] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): fuse6> on 2021-07-19T00:02:00+00:00
[2021-07-20 17:39:40,545] {standard_task_runner.py:52} INFO - Started process 535047 to run task
[2021-07-20 17:39:40,556] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'Fuse', 'fuse6', '2021-07-19T00:02:00+00:00', '--job-id', '309', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpdoeygatc', '--error-file', '/tmp/tmpbi93yqgd']
[2021-07-20 17:39:40,557] {standard_task_runner.py:77} INFO - Job 309: Subtask fuse6
[2021-07-20 17:39:41,712] {logging_mixin.py:104} INFO - Running <TaskInstance: Fuse.fuse6 2021-07-19T00:02:00+00:00 [running]> on host deepika-Inspiron-5570
[2021-07-20 17:39:41,990] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=sthadpka93@gmail.com
AIRFLOW_CTX_DAG_OWNER=postgres
AIRFLOW_CTX_DAG_ID=Fuse
AIRFLOW_CTX_TASK_ID=fuse6
AIRFLOW_CTX_EXECUTION_DATE=2021-07-19T00:02:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-07-19T00:02:00+00:00
[2021-07-20 17:39:48,631] {python.py:151} INFO - Done. Returned value was: None
[2021-07-20 17:39:48,635] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=Fuse, task_id=fuse6, execution_date=20210719T000200, start_date=20210720T115440, end_date=20210720T115448
[2021-07-20 17:39:49,033] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-20 17:39:49,046] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-07-20 18:40:38,611] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Fuse.fuse6 2021-07-19T00:02:00+00:00 [queued]>
[2021-07-20 18:40:38,632] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Fuse.fuse6 2021-07-19T00:02:00+00:00 [queued]>
[2021-07-20 18:40:38,632] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-20 18:40:38,632] {taskinstance.py:1068} INFO - Starting attempt 1 of 8
[2021-07-20 18:40:38,633] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-20 18:40:38,721] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): fuse6> on 2021-07-19T00:02:00+00:00
[2021-07-20 18:40:38,726] {standard_task_runner.py:52} INFO - Started process 711783 to run task
[2021-07-20 18:40:38,737] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'Fuse', 'fuse6', '2021-07-19T00:02:00+00:00', '--job-id', '309', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpu4nqbj_p', '--error-file', '/tmp/tmpg3ugg6lo']
[2021-07-20 18:40:38,739] {standard_task_runner.py:77} INFO - Job 309: Subtask fuse6
[2021-07-20 18:40:38,897] {logging_mixin.py:104} INFO - Running <TaskInstance: Fuse.fuse6 2021-07-19T00:02:00+00:00 [running]> on host deepika-Inspiron-5570
[2021-07-20 18:40:39,311] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=sthadpka93@gmail.com
AIRFLOW_CTX_DAG_OWNER=postgres
AIRFLOW_CTX_DAG_ID=Fuse
AIRFLOW_CTX_TASK_ID=fuse6
AIRFLOW_CTX_EXECUTION_DATE=2021-07-19T00:02:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-07-19T00:02:00+00:00
[2021-07-20 18:40:48,409] {python.py:151} INFO - Done. Returned value was: None
[2021-07-20 18:40:48,521] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=Fuse, task_id=fuse6, execution_date=20210719T000200, start_date=20210720T125538, end_date=20210720T125548
[2021-07-20 18:40:48,621] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-20 18:40:48,636] {local_task_job.py:151} INFO - Task exited with return code 0
